test/integration/OrderBook.spec.ts

```diff
@@ -30,7 +30,7 @@ describe('OrderBook', () => {
     const p2pRepository = new P2PRepository(loggers.p2p, db.models);
 
     await p2pRepository.addNode(
-      { nodePubKey: nodeKey.nodePubKey, addresses: [] },
+      { nodePubKey: nodeKey.nodePubKey, addresses: [], lastAddress: { host: '', port: 0 } },
```

Comment:
Can we not just leave `lastAddress` as `undefined` here?

Reply:
since I'll change it as you described `lastAddress?` I guess it'll be better to remove it there.